id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/helper/CredentialIssueRequestHelper.java

```diff
@@ -0,0 +1,47 @@
+package io.mosip.credential.request.generator.helper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.mosip.credential.request.generator.entity.CredentialEntity;
+import io.mosip.idrepository.core.dto.CredentialIssueRequestDto;
+import io.mosip.idrepository.core.dto.CredentialServiceRequestDto;
+import io.mosip.idrepository.core.logger.IdRepoLogger;
+import io.mosip.kernel.core.logger.spi.Logger;
+import io.mosip.kernel.core.util.CryptoUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CredentialIssueRequestHelper {
+    private static final Logger LOGGER = IdRepoLogger.getLogger(CredentialIssueRequestHelper.class);
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private io.mosip.credential.request.generator.util.CryptoUtil cryptoUtil;
+
+    public CredentialServiceRequestDto getCredentialServiceRequestDto(CredentialIssueRequestDto credentialIssueRequestDto, String requestId) throws JsonProcessingException {
```

Comment:
please add the Junit test case.

Reply:
https://mosip.atlassian.net/browse/MOSIP-41881